src/main/java/mate/academy/hwspringbootintro/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.hwspringbootintro.controller;
+
+import java.util.List;
+import mate.academy.hwspringbootintro.dto.BookDto;
+import mate.academy.hwspringbootintro.dto.CreateBookRequestDto;
+import mate.academy.hwspringbootintro.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
```suggestion
@RequestMapping("/books")
```